features/fig_newton.feature

```diff
@@ -57,18 +64,28 @@ Feature: Functionality of the fig_newton gem
     Given I have read the configuration file
     When I ask for a value that does not exist named "does_not_exist" that has a default value "the default value"
     Then I should see "the default value"
-  
+
+  Scenario: Requesting data that does not exist but has a default value of true should return the default value of true
```

Comment:
This scenario and the one below it are failing in local tests -- looks like my proposed change does not work for booleans specified as defaults. Not sure if this is even necessary, but looking into it.